community/cypher/docs/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/QueryPlanTest.scala

```diff
@@ -195,9 +195,9 @@ class QueryPlanTest extends DocumentingTestBase {
       title = "Cartesian Product",
       text =
         """Produces a cross product of the two inputs -- each row coming from the left child will be matched with all the rows from the right child operator.
-          |The following query will join all the people with all the locations and return the cartesian product of the nodes with those labels.
+          |The following query will join all the people with all the teams and return the cartesian product of the nodes with those labels.
```

Comment:
+1 on getting rid of "join".

Do we know any typical case where users end up with a Cartesian Product by mistake.
I'm wondering if we could write something helpful around that.